sdk/src/transaction_context.rs

```diff
@@ -48,8 +48,8 @@ pub struct TransactionContext {
     account_touched_flags: RefCell<Pin<Box<[bool]>>>,
     instruction_context_capacity: usize,
     instruction_stack: Vec<usize>,
-    number_of_instructions_at_transaction_level: usize,
-    instruction_trace: InstructionTrace,
+    _number_of_instructions_at_transaction_level: usize,
```

Comment:
I assume this is left here due to ABI compatibility? Can you add a comment explaining that?

Reply:
No, good finding. The field can be removed (for now).